Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -1739,34 +1707,6 @@ def send_alert_attachment_command(
     )
 
 
-def get_alert_attachments_command(
```

Comment:
no need to remove it's code, you can just add the deprecated key in the yml file